modules/gcs/outputs.tf

```diff
@@ -48,6 +48,18 @@ output "notification" {
   value       = local.notification ? google_storage_notification.notification[0].self_link : null
 }
 
+output "objects" {
+  description = "Objects in GCS bucket."
+  value = { for k, v in google_storage_bucket_object.objects : k => {
+    crc32c      = v.crc32c
+    md5hash     = v.md5hash
+    self_link   = v.self_link
+    output_name = v.output_name
+    media_link  = v.media_link
+    }
+  }
```

Comment:
Does
```
  value = google_storage_bucket_object.objects
```
work?

Reply:
This is what you get if you do that:

```
│ Error: Output refers to sensitive values
│
│   on main.tf line 100:
│  100: output "bucket-1-objects" {
│
│ To reduce the risk of accidentally exporting sensitive data that was intended to be only internal, Terraform requires that any root module output containing sensitive data be explicitly marked as sensitive, to confirm your intent.
│
│ If you do intend to export this data, annotate the output value as sensitive by adding the following argument:
│     sensitive = true
```

I would assume because this would export any customer encryption key info inputted as well as everything else

Reply:
Ok then what you have is fine